### PR TITLE
En 873 user settings endpoint

### PR DESCRIPTION
**Pull request by stefanopuddu**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/usersettings/UserSettingsService.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        }
+    }
+
+    @Override
+    public UserSettingsDto updateUserSettings(UserSettingsRequest request) {
+        try {
+
+            Map<String, String> params = request.toMap();
+            Map<String, String> systemParams = this.getSystemParams();
+            params.keySet().forEach((param) -> {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/usersettings/UserSettingsService.java]: why not `systemParams.putAll(params)` ?

File: engine/src/main/java/org/entando/entando/web/usersettings/UserSettingsController.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<?> getUserSettings(RestListRequest requestList) {
+        UserSettingsDto userSettings = this.getUserSettingsService().getUserSettings();
+        return new ResponseEntity<>(new RestResponse(userSettings), HttpStatus.OK);
+    }
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(method = RequestMethod.PUT, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<?> updatePageSettings(@Valid @RequestBody UserSettingsRequest request, BindingResult bindingResult) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/usersettings/UserSettingsController.java]: logging in controller methods

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**